### DISCUSSION TITLE
SSL Installation: Where is my root folder?

### DISCUSSION THREAD

**Discussion by Splicho** [requarks/wiki#5106] [Help / Questions]
Hello,

I have installed my wiki.js with one click install @ digitalocean. However, throughout the assignment of an SSL Certificate, it asks me to put in the root directory of my installed wiki.js. 

I am running it on ubuntu/docker. I just can't find the folder.

Does anyone have a clue?

**Comment by imaricodes** [comment]
Just spent an up and down 48hrs with the whole digital ocean / certbot installation. I'm new to the whole cloud VM world... Did you solve this?

I ran into that issue and could not find an answer either. I needed the webroot path because I was trying to run certbot with the '--webroot' command instead of '--standalone' because the --standalone process kept falling because some thing (I could not identify) was using port 80. 

I never figured out the webroot path needed for the --webroot command install because I figured out that docker was blocking port 80. I just had to stop and then restart it to get the certbot to complete running.

Still not out of the woods. My site is still showing 'not secure'. But, at this point it's only been a few hours. I read it could take a while. 

So what happened with the webroot hunt?

**Reply by Splicho** [reply]
Actually, I asked in LetsEncrypt forum, and they told me that Wiki.JS has an automatic SSL feature in the admin menu.
You might try that out.

I run that SSL setup in the admin menu, and it installed the cert for me. It might work for you too.

**Reply by imaricodes** [reply]
Interesting. Don't know why that wasn't obvious to me. My security area shows Let's Encrypt being active. There is nothing clickable in that area. I'm guessing this means it is installed, but my site still shows 'not secure' in the browser. I'll have to go over there and ask some questions
<img width="651" alt="Screen Shot 2022-04-10 at 3 00 53 AM" src="https://user-images.githubusercontent.com/16360390/162606581-43c2a4f9-8b12-44a5-af8d-d260b4c6c0a1.png">
.

**Reply by NGPixel** [reply]
Have you followed the Let's Encrypt guide https://docs.requarks.io/install/digitalocean#automatic-https-with-lets-encrypt ?

The screenshot you posted simply shows the SSL provider as being Let's Encrypt, it is however not enabled or in use (as it only shows the HTTP Port and there's no button to enable Auto-redirect to HTTPS)

**Comment by imaricodes** [comment]
Ok. Got it. I didn't realize those docker commands were for that installation. Kind of skipped over it. I've learned a lot! The messages in the terminal show that the process ran correctly. As of now (about an hour), my site is still not secure, but as I understand it can take a while to show up as encrypted.

**Comment by imaricodes** [comment]
Followed instructions. Everything works now. Great!
thx all
